edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/cloud/CloudListener.cs

```diff
@@ -15,7 +15,7 @@ public CloudListener(IEdgeHub edgeHub, string clientId)
             this.clientId = Preconditions.CheckNotNull(clientId, nameof(clientId));
         }
 
-        public Task<DirectMethodResponse> CallMethodAsync(DirectMethodRequest request) => this.edgeHub.InvokeMethodAsync(this.clientId, request);
+        public Task<DirectMethodResponse> CallMethodAsync(DirectMethodRequest request) => this.edgeHub.InvokeMethodAsync("$upstream", request);
```

Comment:
The first argument to InvokeMethodAsync should be the caller id. Cloud Listener would previously report this incorrectly, as this.clientId is the name of the callee device or module.

Reply:
Nit: upstream